code/modules/jobs/job_types/gimmick.dm

```diff
@@ -69,6 +71,7 @@
 	access = list(ACCESS_MAINT_TUNNELS)
 	minimal_access = list(ACCESS_MAINT_TUNNELS)
 	gimmick = TRUE
+	warning = "You are a paranoid maintenance dweller, this job is not a license to grief."
```

Comment:
Obviously you can't grief as a normal role, would make more sense as something telling them not to go too far. (Although debtor should be removed anyway)